src/flows/parser_item_html_backfill_filesplit_flow.py

```diff
@@ -61,14 +39,14 @@ def split_file(key: str) -> (str, List[pd.DataFrame]):
     df_iterator = pd.read_csv(
         response['Body'], escapechar='\\', sep="|", header=None, usecols=[0, 1], chunksize=STAGE_CHUNK_ROWS,
         names=['resolved_id', 'compressed_html'])
-    return df_iterator
+    return list(enumerate(df_iterator))
```

Comment:
This will materialize the entire 6 GB file in memory as a dataframe. Didn't that cause problems before?